src/test/static/project-with-errors/build/output.html

```diff
@@ -0,0 +1 @@
+<link rel="import" href="built-nonexistant-import.html">
```

Comment:
There are existing tests that analyze the packages at test/static/project and test/static/project-with-errors and assert on the documents found within. By adding these files we're getting assertions that they're not part of those packages